UnitTesting/DependencyInjectionFixture.cs

```diff
@@ -0,0 +1,36 @@
+﻿using DataAccess;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Services.Interfaces;
+using Services;
+using Domain;
+
+namespace UnitTesting
+{
+    public class DependencyInjectionFixture
+    {
+        public ServiceProvider ServiceProvider { get; private set; }
+        private string connectionString = "Server=localhost;Database=DotNetCourseDatabase;TrustServerCertificate=true;Trusted_Connection=true;";
+
+        public DependencyInjectionFixture()
+        {
+            var serviceCollection = new ServiceCollection();
+            ConfigureServices(serviceCollection);
+            ServiceProvider = serviceCollection.BuildServiceProvider();
+        }
+
+        private void ConfigureServices(IServiceCollection services)
```

Comment:
Do you also take care of database migrations anywhere? You should not rely on the database and the schema already existing just to run the tests.